docs/source/api/parameters_id_unassigned_profiles.rst

```diff
@@ -18,6 +18,7 @@
 *****************************************
 ``parameters/{{ID}}/unassigned_profiles``
 *****************************************
+.. deprecated:: ATCv4
 .. warning:: There are **very** few good reasons to use this endpoint - be sure not limit said use.
```

Comment:
Can you do me a favor and fix the typo here? That "not" is meant to be "to".

Reply:
👍